### ISSUE TITLE
Compressed PDB files are no longer available

### ISSUE THREAD

**Issue by srpape** [google/rekall#301]
Rekall 1.6.0 fails to "fetch_pdb", because it attempts to download the compressed version of the symbol file.

I contacted Microsoft about the "pd_" files 404'ing, and received this reply:

```
We’ve made some changes to our symbol server recently and are intentionally no longer server compressed symbols.
````
 
I believe this means that, going forward, Rekall should attempt to download the pdb file itself, rather than a compressed version.

**Comment by scudette** [comment]
This should be fixed by 60011c432187dd54525d2c9df41ae3f214f800ee.

We will need to make a new release soon to make it more widely available.

**Comment by scudette** [comment]
I think this went out with the previous release.
